packages/deployment/upgrade-test/Dockerfile

```
@@ -81,7 +81,14 @@ ARG BOOTSTRAP_MODE
 ENV THIS_NAME=agoric-upgrade-10 USE_JS=1 BOOTSTRAP_MODE=${BOOTSTRAP_MODE}
 
 WORKDIR /usr/src/agoric-sdk/
-COPY ./bash_entrypoint.sh ./env_setup.sh ./start_to_to.sh ./package.json ./*.mjs ./upgrade-test-scripts/
+COPY ./bash_entrypoint.sh ./env_setup.sh ./start_to_to.sh ./package.json ./*.js ./upgrade-test-scripts/
+
+WORKDIR /usr/src/agoric-sdk/upgrade-test-scripts
+RUN yarn
+RUN echo '. /usr/src/agoric-sdk/upgrade-test-scripts/env_setup.sh' >> ~/.bashrc 
+
+WORKDIR /usr/src/agoric-sdk/
+
```

these lines complicate the boilerplate. Without templating it's hard to see what's supposed to be part of each layer vs what's specific to the current.

Would the following work?
```suggestion
RUN cd upgrade-test-scripts && yarn
RUN echo '. /usr/src/agoric-sdk/upgrade-test-scripts/env_setup.sh' >> ~/.bashrc 
```

Also why does every layer need to append to `~/.bashrc`? Shouldn't doing it once at the top suffice? Hmm, ditto for the yarn install.

---

Each build stage is seperate from each other, so the layers we add in `agoric-upgrade-10` won't be present in `agoric-upgrade-11`, which is why we have to do this for every stage.

---

Oh, I thought they were layered upon each other. Still haven't grokked this scheme